scripts/minikube.sh

```diff
@@ -55,7 +55,17 @@ function install_minikube() {
         read -ra mk_version <<<"$(${minikube} version)"
         version=${mk_version[2]}
         echo "minikube already installed with ${version}"
-        return
+        mku_version=$(minikube update-check 2> /dev/null || ret=${?})
```

Comment:
you need to execute `${minikube}` like done in other places.

`ret` is set in a subshell (started with `$(...)`), I would be surprised if the value of `ret` is stored in `ret` after the subshell exits. Please check.

Reply:
> you need to execute `${minikube}` like done in other places.

Sure.

> 
> `ret` is set in a subshell (started with `$(...)`), I would be surprised if the value of `ret` is stored in `ret` after the subshell exits. Please check.

Yes, it will. The trick was played here because simply using:
mku_version=$(minikube update-check)
will exit the whole script if the `minikube update-check` command fails, like in case the network is not available etc..

In the case of failure the above trick will 1. help not exit the script 2. collect the exit code of the command too

Thanks!